go-kit/metrics/provider.go

```diff
@@ -14,13 +16,17 @@ import (
 	"github.com/go-kit/kit/metrics"
 )
 
-// Provider represents all the kinds of metrics a provider must expose. This is
-// here for 2 reasons: (1) go-kit/metrics/provider imports all the providers in
-// the world supported by go-kit cluttering up your vendor folder; and (2)
-// provider.Provider (hmmmmm stutter)!
+// Provider represents the different types of metrics that a provider
+// can expose. We duplicate the definition from go-kit for 2 reasons:
+//
+//  1. A little copying never hurt anyone (and in copying, we avoid the
+//       need to import and vendor all of go-kit's supported providers
```

Comment:
Minor: this indentation is a bit off from the indentation for the second line of 2.

Reply:
no sacrifice too great!